Expecto.Tests/Tests.fs

```diff
@@ -459,6 +459,85 @@ let expecto =
           assertTestFailsWithMsg msg (test, Normal)
       ]
 
+      testList "#distributed" [
+        testCase "identical sequence" <| fun _ ->
+          Expect.distributed [|21;37|] <| Map [
+              (21, 1)
+              (37, 1)
+            ] <| "Identical"
+
+        testCase "sequence contains all in different order" <| fun _ ->
+          Expect.distributed [|21;37|] <| Map[
+              (37, 1)
+              (21, 1)
+            ]
+                             <| "Same elements in different order"
+
+        testCase "sequence doesn't contain repeats in expected" <| fun _ ->
+          let format = "Sequence should contain one, two and four"
+          let test () =
+            Expect.distributed [|2; 2; 4|]  <| Map [
+                  (2, 1)
+                  (1, 1)
+                  (4, 2)
+                ] <| format
+          let msg =
+            sprintf "%s.
+    Sequence `actual` does not contain all `expected` elements.
+        All elements in `actual`:
+        {2, 2, 4}
+        All elements in `expected` ['item', 'number of expected occurances']:
+        {[1, 1], [2, 1], [4, 2]}
+        Missing elements from `actual`:
+        '1' (0/1)\t| -\n\t'4' (1/2)\t| #-
+        Extra elements in `actual`:
+        '2' (2/1)\t| #/"
+              format
+          assertTestFailsWithMsg msg (test, Normal)
+
+        testCase "sequence does contain repeats in expected but should not" <| fun _ ->
+          let format = "Sequence should contain two, two and four"
+          let test () =
+            Expect.distributed [|2; 2|] <| Map[
+                (2, 2)
+                (4, 1)
+            ] <| format
+          let msg =
+            sprintf "%s.
+    Sequence `actual` does not contain all `expected` elements.
+        All elements in `actual`:
+        {2, 2}
+        All elements in `expected` ['item', 'number of expected occurances']:
+        {[2, 2], [4, 1]}
+        Missing elements from `actual`:
+        '4' (0/1)\t| -
+        Extra elements in `actual`:
+        "
+              format
+          assertTestFailsWithMsg msg (test, Normal)
+
+        testCase "sequence does not contains everything expected" <| fun _ ->
+          let format = "Sequence should contain two and two"
+          let test () =
+            Expect.distributed [|2; 2; 4|] <| Map[
+              (2, 1)
+              (4, 1)
+            ] <| format
+          let msg =
+            sprintf "%s.
+    Sequence `actual` does not contain all `expected` elements.
+        All elements in `actual`:
+        {2, 2, 4}
+        All elements in `expected` ['item', 'number of expected occurances']:
+        {[2, 1], [4, 1]}
+        Missing elements from `actual`:
+        
+        Extra elements in `actual`:
+        '2' (2/1)\t| #/"
```

Comment:
I think we can drop the `#/` and #-` ligatures when we have the distribution printed.